src/tests/conftest.py

```diff
@@ -129,6 +129,16 @@ def filled_router(
     return router
 
 
+@pytest.fixture()
+def reloadrouter(filled_router):
+    """The filled_router, but reloaded before being used.
+    this is the Nick-trick :-)  Needed when schemas
+    with relations are used.
+    """
+    filled_router.reload()
```

Comment:
Misschien `router.reload()` binnen de `filled_router` werkt ook goed?

Reply:
Ja, heb ik ook overwogen. Als die reload snel genoeg is (ik denk het wel) dan maakt dat het wat simpeler. Ga ik ff proberen.